glfs.c

```diff
@@ -430,58 +430,6 @@ static char* tcmu_get_path( struct tcmu_device *dev)
 	return config;
 }
 
-
-static bool glfs_check_config(const char *cfgstring, char **reason)
-{
```

Comment:
I'm not sure if it is a good Idea to remove check_config functionality for all handlers ?

IMO, its up to the handlers to choose, for example rbd did not choose to have check_config, may be because the way they ensure to push the target creation (from their wrapper, gwcli) commands will ensure to either create target or rewind the backend creation in case of failure. 

IMHO, for handlers whose fs calls are local, its not a bad idea to have config_check() as they ensure the config string at the first place.

The only reason that I intend to drop config_check for gluster is that, it involves additional network calls and the cost here might not be worthy when the wrapper tool take care of rewinding the backstore creation.

Let me know your thoughts ?